sdk/identity/identity/test/internal/identityClient.spec.ts

```diff
@@ -56,7 +56,11 @@ describe("IdentityClient", function() {
       mockHttp.tokenCredentialOptions
     );
     await assertRejects(credential.getToken("https://test/.default"), (error) => {
-      assert.equal(error.name, "CredentialUnavailableError");
+      if (isNode) {
+        assert.equal(error.name, "CredentialUnavailableError");
+      } else {
+        assert.equal(error.name, "AuthenticationError");
```

Comment:
This is the same behavior in v1 also right?

Reply:
i.e. the fact that error that user would get is different in node vs browser for the same test case?

Reply:
Let me check

Reply:
Yes: https://github.com/Azure/azure-sdk-for-js/blob/hotfix/identity_1.3.0/sdk/identity/identity/test/internal/identityClient.spec.ts#L58

Reply:
The link suggests that previously we had `AuthenticationError` thrown for node as well. The reason for difference in node?

Reply:
This test uses `ClientSecretCredential`, which now has two different implementations. In this case the Node one is correct because the provided tenant is not a valid tenant, so this credential can't be used. MSAL does validations like this.